src/js/include/render.js

```diff
@@ -44,6 +44,20 @@ export function initResults(itemsToScan, activeOnly) {
 }
 
 export function updateResult(response, job) {
+  const report = response.reports?.phpcs_phpcompatibilitywp?.report;
+
+  // If we have a success response but don't have report data, show an error.
+  if (!report) {
+    updateResultFailure(
+      {
+        status: "failed",
+        message: __("No scan results found", "wpe-php-compat"),
```

Comment:
This message is a part of the reporting, which is included to the raw text.

Since all error messages which come from wptide API are in English, isn't it good to keep non-translated strings for all hardcoded error messages too?

We already have many non-translated messages for raw reports: "Compatible with PHP {version}", "PHP {version} incompatibilities", "Scan status: {status}", etc.

I understand that this particular message will be also displayed in the "item results box", so it might be considered as a part of the UX potentially to translate.

But the target audience of this plugin are mostly developers, who are able to read English even if it's not a language of the website. And all other errors which come from the API will be in English. And sharing raw reports between developers is better in English.